intbot/core/bot/channel_router.py

```diff
@@ -0,0 +1,97 @@
+"""
+Channel router that decides where to send particular message
+"""
+
+from dataclasses import dataclass
+
+from core.integrations.github import (
+    GithubProjects,
+    GithubRepositories,
+    parse_github_webhook,
+)
+from core.models import Webhook
+from django.conf import settings
+
+
+@dataclass
+class DiscordChannel:
+    channel_id: str
+    channel_name: str
+
+
+dont_send_it = DiscordChannel(channel_id="0", channel_name="/dev/null")
```

Comment:
I'd prefer a better naming here - `null_channel` ?

Reply:
I can think of a better name, but null_channel doesn't quite capture the idea – the idea is to compare to that singleton, and if so, then don't send that message at all.